src/logger.js

```diff
@@ -8,7 +8,10 @@ var parentLogger;
 
 exports.init = function(config) {
   if (config.logger) {
-    parentLogger = config.logger.child({module: 'instana-nodejs-logger-parent'});
+    parentLogger = config.logger;
```

Comment:
If I understand your change correctly, your intention would be to use `config.nonBunyanLogger` to pass in a logger object. Any particular reason to change the above line, handling the Bunyan case?

Reply:
This was changed accidentally. i removed this change.